src/clojure/nrepl/middleware/session.clj

```diff
@@ -3,20 +3,18 @@
   {:author "Chas Emerick"}
   (:require
    clojure.main
+   [nrepl.util.classloader :as classloader]
```

Comment:
Don't forget to sort the namespaces after renaming them. I think the linter might complain about this as well.